17style-elements/index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Travel Disclosure - Destination: Moon</title>
+    <link href="styles.css" rel="stylesheet" />
+  </head>
+  <body>
+    <section class="disclosure-alert">
+      <header class="disclosure-header">
+        <h2 class="disclosure-title">
+          <em>Destination: Moon</em> Travel Disclosure
+        </h2>
+      </header>
+      <div class="disclosure-content">
+        <p>
+          Although space travel is routine practice, there are many unknown
+          possibilities that any traveller must be aware of before traveling
+          with <em>Destination: Moon</em>. Agreeing to this disclosure of knowns
+          is required prior to purchase of tickets with
+          <em>Destination: Moon</em>. PLEASE, READ AND AGREE TO THE FOLLOWING
+          DISCLOSURE OF TRAVEL UNKNOWNS BEFORE PROCEEDING TO PURCHASE.
+        </p>
+        <div class="legal-contents">
+          <p>
+            Effect of Termination. Upon termination, You agree not to use it
+            under the terms of Sections 4(a) through 4(e) for that Covered Code,
+            or any third party. Description of Modifications.
+          </p>
+          <p></p>
```

Comment:
You can delete this empty `p` tag. If you're using it to add a row of whitespace, it would be better to add margin to either paragraph above or below using CSS.